Makefile

```diff
@@ -9,11 +9,11 @@ PIP_COMPILE = pip-compile --output-file ${REQUIREMENTS} setup.py
 ANSIBLE_ENV=COLUMNS=80
 src/commcare_cloud/help_cache/ansible.txt: export ANSIBLE_CONFIG=src/commcare_cloud/ansible/ansible.cfg
 src/commcare_cloud/help_cache/ansible.txt:
-	$(ANSIBLE_ENV) ansible -h > src/commcare_cloud/help_cache/ansible.txt
+	$(ANSIBLE_ENV) ansible -h | sed 's/^options:/optional arguments:/' > src/commcare_cloud/help_cache/ansible.txt
```

Comment:
Would it be disruptive to do the opposite replacement, ending up with `options:`, since that is the format supported by newer versions like Python 3.10? That way this could be removed at some point when we drop support for older versions of Python that output `optional arguments:`.

Reply:
That does make more sense. Will update shortly.

Reply:
See 88f370a999a0d720f5bf99ca35ea980a71d9fe52